packages/cli/src/options/beaconNodeOptions/execution.ts

```diff
@@ -5,13 +5,17 @@ import {ICliCommandOptions, extractJwtHexSecret} from "../../util";
 export type ExecutionEngineArgs = {
   "execution.urls": string[];
   "execution.timeout": number;
+  "retry.attempts": number;
```

Comment:
I think it would be better if we prefix the name space here, `execution.retryAttempts`, `execution.retryDelay`